samples/vision/custom_vision_object_detection_sample.py

```diff
@@ -2,15 +2,16 @@
 import sys
 import time
 
-from azure.cognitiveservices.vision.customvision.training import training_api
+from azure.cognitiveservices.vision.customvision.training.custom_vision_training_client import CustomVisionTrainingClient
```

Comment:
Please change it to:
from azure.cognitiveservices.vision.customvision.training import CustomVisionTrainingClient

I plan to remove the former verbose syntax